src/mesh/RadioInterface.cpp

```diff
@@ -123,6 +123,17 @@ const RegionInfo regions[] = {
     */
     RDEF(MY_919, 919.0f, 924.0f, 100, 0, 27, true, true, false),
 
+    /*
+        Singapore
+        SG_433 Band 26a: 433.05 - 434.79 MHz at 10mW, 10% duty cycle.
+        SG_868 Band 29: 868 - 869 MHz at 500mW, no restrictions.
+        SG_923 Band 30c: 917 - 925 MHz at 100mW, 1% duty cycle.
+        https://www.imda.gov.sg/-/media/imda/files/regulation-licensing-and-consultations/ict-standards/telecommunication-standards/radio-comms/imdatssrd.pdf
+    */
+    RDEF(SG_433, 433.05f, 434.79f, 10, 0, 10, true, false, false),
+    RDEF(SG_868, 868.0f, 869.0f, 100, 0, 27, true, false, false),
+    RDEF(SG_923, 917.0f, 925.0f, 1, 0, 20, true, false, false),
```

Comment:
%1 Duty Cycle is not viable and should not be added.

Reply:
> %1 Duty Cycle is not viable and should not be added.

Would Meshtastic be considered an "LPWAN network access point" or an "LPWAN end device"? If it is the former, I will raise to 500mW at 10% duty cycle. If it is the latter, I will remove the 923MHz band from the PR.

I suppose the 1% was decided based on sensor network sensors. I can direct this question to the regulatory authority otherwise.

Reply:
Meshtastic is not LPWAN where "end devices" communicate with a central "access point" or "gateway".

but I see there is provision for "Non-specific SRD" (30d), which is great. this is a generic category and should be safe to use.